src/Installer/CoreInstaller.php

```diff
@@ -29,8 +29,15 @@
 class CoreInstaller extends LibraryInstaller
 {
     /**
-     * Returns the installation path of a package
-     *
+     * @param  string $packageType
+     * @return bool
+     */
+    public function supports($packageType)
```

Comment:
please create a ticket that describes the concrete case where not having this code change fails.

Reply:
As mentioned above, I´ve simply overlooked the constructor arguments because I did not expect them to be present.